### DISCUSSION TITLE
PR splitting -> Update Bonanza

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#321] [Ideas]
Hello,

As said before, to take care of this (now enormous) PR that we have there, I suggest to split the PR into different country PRs.

To prevent having all of them at once (and so, making it a bit more difficult to deal with, since we're not always available, I suggest we make this : 

4 PRs will be made per region (let's do it alphabetically, 4 Asia & MENA, 4 EU, 4 NA and then 4 SA)

Once one set of PR is approved, another one takes in, and so it'll make it easier for everybody to take care of the PRs.

If it's alright for you, then we'll begin once everyone's approval is given.

**Comment by MapGuy11** [comment]
Sounds good!

**Comment by freearhey** [comment]
The discussion has been moved to _iptv-org/iptv-org.github.io_ repository due to GitHub's decision to disable "Team Discussions": https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/
